### ISSUE TITLE
[Bug] Visual Bug on Pokedex when Randomizing Emerald

### ISSUE THREAD

**Issue by ultimatetemper** [upr-fvx/universal-pokemon-randomizer-fvx#12]
**Describe the bug**
When opening the pokedex on a randomized Emerald, the preview doesn't appear as usual.

**To Reproduce**
1. Randomize an Emerald Game
2. Open Pokedex
3. See error

**Expected behavior**
See screenshots.

**Screenshots**
Correct behaviour:
![randomdexcorrect](https://github.com/user-attachments/assets/239e7efd-9d1b-43a3-8570-2d93e4d23da0) 
Current behaviour:
![randomdexbug](https://github.com/user-attachments/assets/480bc136-1a1f-4333-b5c8-f2c073a29de1) 

**System (please complete the following information):**
 - Operating System: Windows 10 x64
 - Java Version: 8 update 411
 - Randomizer Version: FVX 1.0.2

**Additional context**
I have to check if this happens on Ruby and Sapphire as well.

**Comment by voliol** [comment]
Thank you for reporting :). 
Do you remember what settings you used when randomizing? Knowing those helps a lot when debugging. Also, is this the Spanish version of Emerald? The randomizer should be able to work with all official versions, so that's not a problem in itself, just another detail that matters when debugging.

**Comment by ultimatetemper** [comment]
Yes, it's the Spanish version of Emerald. I'll test it on the English versions as well.
For the settings I used, should I upload the .rnqs file I used on this?

**Comment by voliol** [comment]
The .rnqs file would be great, yes :)

**Comment by ultimatetemper** [comment]
Here it is (I couldn't upload the .rnqs file directly so I zipped it)
[randomizer settings.zip](https://github.com/user-attachments/files/17105245/randomizer.settings.zip)

**Comment by voliol** [comment]
Bug fixed for the next version.
